src/eosjs-serialize.ts

```diff
@@ -1143,3 +1175,173 @@ export function deserializeAction(contract: Contract, account: string, name: str
         data: deserializeActionData(contract, account, name, data, textEncoder, textDecoder),
     };
 }
+
+function addAdditionalTypes(): Map<string, Type> {
+    const initialTypes = createInitialTypes();
+    initialTypes.set('null_t', createType({
+        name: 'null_t',
+        serialize(buffer: SerialBuffer, anyvar: Anyvar) {}, // tslint:disable-line no-empty
+        deserialize(buffer: SerialBuffer, state?: SerializerState) {} // tslint:disable-line no-empty
+    }));
+    initialTypes.set('any_object', createType({
+        name: 'any_object',
+        serialize: serializeAnyObject,
+        deserialize: deserializeAnyObject
+    }));
+    initialTypes.set('any_array', createType({
+        name: 'any_array',
+        serialize: serializeAnyArray,
+        deserialize: deserializeAnyArray
+    }));
+    return initialTypes;
+}
+
+const additionalTypes = addAdditionalTypes();
+
+const anyvarDefs = {
+    null_t: { index: 0, useShortForm: true, type: additionalTypes.get('null_t') },
+    int64: { index: 1, useShortForm: false, type: additionalTypes.get('int64') },
+    uint64: { index: 2, useShortForm: false, type: additionalTypes.get('uint64') },
+    int32: { index: 3, useShortForm: true, type: additionalTypes.get('int32') },
+    uint32: { index: 4, useShortForm: false, type: additionalTypes.get('uint32') },
+    int16: { index: 5, useShortForm: false, type: additionalTypes.get('int16') },
+    uint16: { index: 6, useShortForm: false, type: additionalTypes.get('uint16') },
+    int8: { index: 7, useShortForm: false, type: additionalTypes.get('int8') },
+    uint8: { index: 8, useShortForm: false, type: additionalTypes.get('uint8') },
+    time_point: { index: 9, useShortForm: false, type: additionalTypes.get('time_point') },
+    checksum256: { index: 10, useShortForm: false, type: additionalTypes.get('checksum256') },
+    float64: { index: 11, useShortForm: false, type: additionalTypes.get('float64') },
+    string: { index: 12, useShortForm: true, type: additionalTypes.get('string') },
+    any_object: { index: 13, useShortForm: true, type: additionalTypes.get('any_object') },
+    any_array: { index: 14, useShortForm: true, type: additionalTypes.get('any_array') },
```

Comment:
More types were added since this list was generated

Reply:
Added the new types